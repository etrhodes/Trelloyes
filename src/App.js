import './App.css';
import './store.js';

function App() {
  return (
    <div>
      
    </div>
  );
}

export default App;
