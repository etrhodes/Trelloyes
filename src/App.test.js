  
import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';
import Card from './Card';
import List from './List';
import STORE from './store';



it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});



describe('Card', () => {
  it('renders the UI as expected', () => {
    const tree = renderer
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Card title="Hello" />)
      .toJSON();
      expect(tree).toMatchSnapshot();
  });
});


describe('List', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<List header="header" />)
      .toJSON();
      expect(tree).toMatchSnapshot();
  });
})

