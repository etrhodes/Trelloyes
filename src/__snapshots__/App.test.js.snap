// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    Delete
  </button>
  <h3>
    Hello
  </h3>
  <p />
</div>
`;
